lexibank_yanglalo.py

```diff
@@ -1,82 +1,72 @@
+            idx = concept.id.split('-')[-1]+'_'+slug(concept.english)
+            args.writer.add_concept(
+                    ID=idx,
+                    Name=concept.english,
+                    Concepticon_ID=concept.concepticon_id,
+                    Concepticon_Gloss=concept.concepticon_gloss,
+                    Chinese_Gloss = concept.attributes['chinese']
+                    )
+            concept_lookup[concept.attributes['number_in_source']] = idx
 
-            languages = {k['Source_ID']: k['ID'] for k in self.languages}
-            ds.add_languages()
-            
-            concepts = {c.attributes['number_in_source']: c.id for c in
-                    self.conceptlist.concepts.values()}
-            for concept in self.conceptlist.concepts.values():
-                ds.add_concept(
-                        ID=concept.id,
-                        Name=concept.english,
-                        Concepticon_ID=concept.concepticon_id,
-                        Concepticon_Gloss=concept.concepticon_gloss,
-                        Chinese_Gloss = concept.attributes['chinese']
-                        )
-            # add lexemes
-            for cogid, entry in pb(enumerate(raw_entries), desc="cldfify",
-                    total=len(raw_entries)):
-                # get the parameter frm the number in source, skipping over
-                # non-published data
-                parameter = entry["Parameter"].split('.')[0]
-                if parameter:
-                    for language in languages:
-                        for row in ds.add_lexemes(
-                            Language_ID=languages[language],
-                            Parameter_ID=concepts[parameter],
+        for cogid, entry in pb(enumerate(raw_entries), desc="cldfify",
+                total=len(raw_entries)):
+            # get the parameter frm the number in source, skipping over
+            # non-published data
+            parameter = entry["Parameter"].split('.')[0]
+            if parameter in concept_lookup:
+                for language in language_lookup:
+                    for row in args.writer.add_forms_from_value(
+                            Language_ID=language_lookup[language],
+                            Parameter_ID=concept_lookup[parameter],
                             Value=entry[language],
                             Source=["Yang2011Lalo"],
-                            Cognacy=cogid,
-                        ):
-                            ds.add_cognate(
-                                lexeme=row,
-                                Cognateset_ID=cogid,
-                                Source=["Yang2011Lalo"],
-                                Alignment_Source="",
+                            Cognacy=cogid
+                            ):
+                        args.writer.add_cognate(
+                            lexeme=row,
+                            Cognateset_ID=cogid+1,
```

Comment:
I suspect, enumerate starts with 0, we don't want a 0 as an ide for a cognate, right?